Aplx2 osu! keyboard (#8847)
qmk/qmk_firmware: keyboards/handwired/aplx2/README.md

# Aplx2

> ![Front](https://i.imgur.com/cN4WfaQ.jpg)  

Reviewer: ```suggestion
![Front](https://i.imgur.com/cN4WfaQl.jpg)  
```

   
A simple 2% keyboard for osu!. Designed to be assembled as a sandwich with a Pro Micro, using M3 x 16mm screws, your choice of MX- or Alps-compatible switches, and 1N4148 diodes.   

**Layout:**  
The default layout is: Z , X .

> Used pins can be found in `config.h`. All ProMicro pins can be found [here](https://golem.hu/article/pro-micro-pinout/).

Reviewer 2: Personally, I'd prefer this link, as it's from SparkFun:
```suggestion
Used pins can be found in `config.h`. All ProMicro pins can be found [here](https://cdn.sparkfun.com/datasheets/Dev/Arduino/Boards/ProMicro16MHzv1.pdf).
```

Author: sure why not!

* Keyboard Maintainer: [Aplyard](https://github.com/Aplyard)
* Hardware Supported: Pro Micro (ATmega32U4)
* Hardware Availability: [GitHub](https://github.com/Aplyard/Aplx2), [Thingiverse](https://www.thingiverse.com/thing:4294608)

Make example for this keyboard (after setting up your build environment):

>      make aplx2:default

Reviewer 2: ```suggestion
     make handwired/aplx2:default
```
